### ISSUE TITLE
Zowe Conformance Badge Revision

### ISSUE THREAD

**Issue by RASakach** [zowe/community#2321]
Revise the Zowe Conformance badges to eliminate references to conformance achievements associated with Zowe releases  that are no longer supported, this includes references to:

-  2019
- Zowe v1

In the example card is listed below, the first 3 ZOWE CONFORMANT entries would be removed.

![Image](https://github.com/user-attachments/assets/3ab10eef-6f24-44f9-9eac-74a176497229)

**Comment by slandath** [comment]
@RASakach You can assign it to me

**Comment by slandath** [comment]
@RASakach Here's a preview of the updated landscape:  [Zowe Conformance](https://deploy-preview-866--omp-landscape.netlify.app/card-mode?category=app-framework-zowe-v3,cli-zowe-v3,api-mediation-layer-zowe-v3,explorer-for-visual-studio-code-zowe-v3,comprehensive-zowe-v3,explorer-for-visual-studio-code-zowe-v3,explorer-for-intelli-j-zowe-v3,api-mediation-layer-zowe-v3,app-framework-zowe-v3,cli-zowe-v3,client-sdk-node-js-zowe-v3&grouping=category)

**Comment by RASakach** [comment]
Looks great Tom!  Thanks for your help.

**Comment by jmertic** [comment]
Merged in updates - https://github.com/zowe/community/issues/2321

**Comment by RASakach** [comment]
@slandath while reviewing the latest landscape, we realized the "ZOWE" extensions cards were not updated to reflect these changes:

![Image](https://github.com/user-attachments/assets/4ba49c63-7a28-4bad-bf84-a80641bbf5df)

When you have a chance, can you also update the ZOWE cards to:

1. eliminate the outdated conformance program references (2019 and V1)
2. reverse the order of the conformance achievements (list V3 at the top, then V2 beneath).

Many thanks for your help in this effort!

**Comment by slandath** [comment]
@RASakach Not a problem.  Should I also remove the "Rocket", "IBM" and "Phoenix" V1 and 2019 badges?

**Comment by RASakach** [comment]
@slandath YES!  Apologies for any confusion.  This change is meant to be applicable for ALL cards - all badges.  Which makes me realize I should send out a notice to all Zowe Extenders making them aware of the change.

**Comment by slandath** [comment]
@RASakach Here's a preview of the updated landscape:   [Zowe Conformance](https://deploy-preview-870--omp-landscape.netlify.app/card-mode?category=zowe-conformant,zowe-conformant-support-provider)

**Comment by RASakach** [comment]
@slandath looks good!  Thank you for making the changes so quickly.
